Solutions/Corvus.Identity.Examples.AzureFunctions/UseMicrosoftRestFunction.cs

```diff
@@ -20,26 +20,50 @@ namespace Corvus.Identity.Examples.AzureFunctions
     /// </summary>
     public class UseMicrosoftRestFunction
     {
-        private readonly UseWebAppManagementWithOldSdk client;
+        private readonly UseWebAppManagementAsServiceIdentityWithOldSdk serviceIdClient;
         private readonly ILogger<UseMicrosoftRestFunction> logger;
         private readonly ExampleSettings settings;
+        private readonly UseWebAppManagementWithIdentityFromConfigWithOldSdk configIdClient;
 
         /// <summary>
         /// Creates a <see cref="UseMicrosoftRestFunction"/>.
         /// </summary>
         /// <param name="settings">Example settings.</param>
-        /// <param name="client">
-        /// The client wrapper for accessing the key vault.
+        /// <param name="serviceIdClient">
+        /// The client wrapper for accessing the key vault using the service identity.
+        /// </param>
+        /// <param name="configIdClient">
+        /// The client wrapper for accessing the key vault using a configured identity.
         /// </param>
         /// <param name="logger">Logger.</param>
         public UseMicrosoftRestFunction(
             ExampleSettings settings,
-            UseWebAppManagementWithOldSdk client,
+            UseWebAppManagementAsServiceIdentityWithOldSdk serviceIdClient,
+            UseWebAppManagementWithIdentityFromConfigWithOldSdk configIdClient,
             ILogger<UseMicrosoftRestFunction> logger)
         {
-            this.client = client ?? throw new System.ArgumentNullException(nameof(client));
+            this.serviceIdClient = serviceIdClient ?? throw new System.ArgumentNullException(nameof(serviceIdClient));
```

Comment:
Fully qualified exception name again...

Reply:
Will fix in https://github.com/corvus-dotnet/Corvus.Identity/issues/214